examples/master.js

```diff
@@ -315,12 +324,49 @@ function onPeerConnectionFailed(remoteClientId, printLostConnectionLog = true, h
     }
 }
 
+/**
+ * Fetches ICE servers, caching them to prevent redundant API calls.
+ * If the cached TURN servers are still valid, it returns them instead of making a new request.
```

Comment:
Could all the form values be included in the docstring? For example:

param {boolean}formValues.sendSrflxCandidates - Server reflexive candidates setting

Reply:
Done